lib/msal-browser/src/interaction_client/SilentIframeClient.ts

```diff
@@ -93,15 +93,16 @@ export class SilentIframeClient extends StandardInteractionClient {
             );
         }
 
-        // Check that prompt is set to none or no_session, throw error if it is set to anything else.
+        // Check the prompt value
         if (
             request.prompt &&
             request.prompt !== PromptValue.NONE &&
             request.prompt !== PromptValue.NO_SESSION
         ) {
-            throw createBrowserAuthError(
-                BrowserAuthErrorCodes.silentPromptValueError
+            this.logger.verbose(
+                `SilentIframeClient. Removing invalid prompt ${request.prompt}`
             );
+            delete request.prompt;
```

Comment:
Objects are passed by reference so this can have unintended effects elsewhere - I suggest removing this and changing the logic below to `prompt: request.prompt === PromptValue.NO_SESSION ? request.prompt : PromptValue.NONE`

Reply:
Alternatively you can spread the input object into a new object at the top of the function and then manipulate the new one as you wish

Reply:
Good point. Fixed